app/src/main/kotlin/com/simplemobiletools/smsmessenger/activities/ThreadActivity.kt

```diff
@@ -1059,4 +1069,13 @@ class ThreadActivity : SimpleActivity() {
 
         setupAdapter()
     }
+
+    private fun updateMessageType() {
+        val settings = getSendMessageSettings()
+        val text = thread_type_message.text.toString()
+        val isGroupMms = participants.size > 1 && config.sendGroupMessageMMS
+        val isLongMmsMessage = getNumPages(settings, text) > settings.sendLongAsMmsAfter && config.sendLongMessageMMS
+        val res = if (attachmentSelections.isNotEmpty() || isGroupMms || isLongMmsMessage) R.string.mms else R.string.sms
```

Comment:
split this into multiple lines too and rename "res" to "stringId" or so

Reply:
[right?](https://github.com/SimpleMobileTools/Simple-SMS-Messenger/pull/363/commits/0195af61efd6bbbe90478311d6f3dd2c37aa9ced)